docs/apache-airflow/concepts/dags.rst

```diff
@@ -363,6 +368,60 @@ However, this is just the default behaviour, and you can control it using the ``
 * ``none_skipped``: No upstream task is in a ``skipped`` state - that is, all upstream tasks are in a ``success``, ``failed``, or ``upstream_failed`` state
 * ``always``: No dependencies at all, run this task at any time
 
+.. note::
```

Comment:
I think this should be a separate chapter. It appears a bit "out of the blue" here and it's not entirely clear why it is here.

I think it could be an "Example of implementing watcher pattern with rules" (or something like that). also I think it should be accompanied with a nice graph image for the DAG and some "overview" on that showing which rules are which - the visual representation of it will make world of a difference here.

Reply:
Please take a look and say what do you think about it now.